src/components/status-header.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC } from 'react'
+import { View, Text} from 'react-native'
+import { MaterialCommunityIcons } from '@expo/vector-icons'
+import SvgIcon from './svg-icons'
+
+type StatusHeaderTypes = 'sent'|'accepted'|'completed'|'give-help'|'receive-help'
+
+interface StatusHeaderProps {
+    type: StatusHeaderTypes
+    color: string
+    hideStatusText?: boolean
+}
+
+interface StatusHeaderParameters{
+    icon: string
+    statusText: string
+}
+
+const statusHeaderInfo: {[key in StatusHeaderTypes]: StatusHeaderParameters} = {
+    'sent': {icon: 'clock-outline', statusText: 'Your task has been sent.\nWaiting for someone to accept it.'},
+    'accepted':  {icon: 'cart-outline', statusText: 'Your task has been accepted.'},
+    'completed':  {icon: 'check-circle-outline', statusText: 'Your task has been completed!'},
+    'give-help':  {icon: '', statusText: ''}, 
+    'receive-help':  {icon: '', statusText: ''}
+}
+
+const StatusHeader: FC<StatusHeaderProps> = (props) => {
+    const renderIcons = () => {
+        if(props.type === 'give-help' || props.type === 'receive-help'){
```

Comment:
Is there any way to make this more typesafe, like looking at the SvgTypes in svg-icon.tsx? I've tried but can't get it to work :( @AdamJonsson

Reply:
Hmm, it seems to be pretty hard to check variable value with a type. This can possibly work: https://stackoverflow.com/a/43621735/4425071
The risk is that the code gets a little "over-engineered"? But if it makes the code easier to handle in the future, then: 👍

Reply:
Yeah that was what I thought as well, I think it's over-engineering right now, but if we are going to expand on it later I think we should do this!